init.lua

```diff
@@ -162,6 +162,8 @@ end
 
 local function get_texture(player)
 	local player_meta = player:get_meta()
+	if player_meta == nil then return "" end
```

Comment:
This is caused by the mod's own fault. By the time this function is called in `minetest.after`, the player might already have logged out. Please at least mention this in both places like so: (suggestion)
```Lua
if not player_meta then
	-- The player left since the last callback and presence check
	return
end
```

Reply:
Add comments, and added two more checks.